sysconfig/sysconfig.go

```diff
@@ -74,6 +80,17 @@ func ApplyFilesystemOnlyDefaults(rootDir string, defaults map[string]interface{}
 	return ApplyFilesystemOnlyDefaultsImpl(rootDir, defaults, options)
 }
 
+// ApplyPreinstallFilesystemOnlyDefaults applies (via configcore.preinstallFilesystemOnlyApply)
+// filesystem modifications under rootDir, according to the defaults.
```

Comment:
this is a bit unclear, as far as I understand rootDir will have to point to the recovery partition at which point maybe it needs a different name and this help needs to be clearer on this?

Reply:
see my other response above